org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java

```diff
@@ -43,8 +43,10 @@
 
 	static final String VERBOSE = "verbose";
 	private static final String TRACE = "trace";
-	private static final String RUNTIME_EXECUTABLE = "runtimeExecutable";
+	public static final String RUNTIME_EXECUTABLE = "runtimeExecutable";
 	private static final String SOURCE_MAPS = "sourceMaps";
+	public static final String CHROMIUM = "Chromium";
+	public static final String CHROME = "Chrome";
```

Comment:
Should those value be the actual `chromium-browser` vs `google-chrome-stable` ?
And the attribute could be a free text (or an editable combo) so people can pass another variation of the runtimeExecutable (can be for instance another path to a local Chromium build).

Reply:
Yes I totally agree on both those points.
Months ago I wanted the user to be able to set the location of their Firefox executable, so this could encompass that feature for both Firefox and Chrome.